src/types/onyx/Card.ts

```diff
@@ -1,9 +1,16 @@
 import {ValueOf} from 'type-fest';
 import CONST from '../../CONST';
 
+type State =
```

Comment:
No `enum` for this?

Reply:
Looks like we ban enum https://github.com/Expensify/App/blob/main/contributingGuides/TS_STYLE.md#enum-vs-union-type